skyfield/elementslib.py

```diff
@@ -9,17 +9,28 @@
                    sinh, sqrt, arccos, arctanh, zeros_like, ones_like, divide,
                    where, pi, cross)
 
-def osculating_elements_of(position, reference_frame=None):
+def osculating_elements_of(position, reference_frame=None, gm_km3_s2=None):
```

Comment:
In the docstring, specify whose GM this is — I think you mean for it to be the sum of the GM of the two bodies, but I think users might be confused and think they're only supposed to provide the GM of the body itself not the center? (And I do like the name! It's very explicit, and "GM" seems very popular among astronomy documentation that I see, which is why I used it elsewhere.)